### PR TITLE
Virt manager cpu fixes

### PR DESCRIPTION
**Pull request by jfehlig**
### PR creator: Description

Improve the virt-manager CPU and memory sections in the VT guide. virt-manager no longer supports specifying current and maximum number of vCPUs. The doc should be updated to reflect that.

Also add descriptions of the host-model and host-passthrough models since they are commonly used and a source of confusion.

### PR creator: Are there any relevant issues/feature requests?

* bsc#1188299 (see comments 6&7)

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [x] SLE 15 SP4/openSUSE Leap 15.4 *(current `main`, no backport necessary)*
  - [x] SLE 15 SP3/openSUSE Leap 15.3
  - [ ] SLE 15 SP2/openSUSE Leap 15.2
  - [ ] SLE 15 SP1
  - [ ] SLE 15 GA
- SLE 12
  - [ ] SLE 12 SP5
  - [ ] SLE 12 SP4
  - [ ] SLE 12 SP3

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

**Comment by jfehlig** [comment]
Thanks @hw-claudio for the comments. I've fixed the typos and the duplicated commit messages in 4/4. Not sure how I managed to have the same commit message in patches 3 and 4.

File: xml/libvirt_configuration_gui.xml [hunk 1]
```diff
@@ -174,49 +174,44 @@
     </mediaobject>
    </figure>
    <para>
-    In the <guimenu>CPUs</guimenu> section, you can configure several
-    parameters related to the number of allocated CPUs.
+    In the <guimenu>CPUs</guimenu> section, you can configure the number
+    of virtual CPUs allocated to the &vmguest;.
+    <term><guimenu>Logical host CPUs</guimenu></term> shows the number of
```
Comment (tbazant) [inline xml/libvirt_configuration_gui.xml:179]:
```suggestion
    <guimenu>Logical host CPUs</guimenu> shows the number of
```

File: xml/libvirt_configuration_gui.xml [hunk 2]
```diff
@@ -174,49 +174,44 @@
...
    <para>
     The <guimenu>Configuration</guimenu> section lets you configure the CPU
     model and topology.
    </para>
    <para>
     When activated, the <guimenu>Copy host CPU configuration</guimenu> option
-    uses the host CPU model for &vmguest;. Otherwise you need to specify the
-    CPU model from the drop-down box.
+    uses the host CPU model for &vmguest;. Details of the host CPU model can be
+    seen in the output of 'virsh capabilities'. When deactivated, the CPU model
```
Comment (tbazant) [inline xml/libvirt_configuration_gui.xml:189]:
```suggestion
    uses the host CPU model for &vmguest;. You can see the details of the host CPU model in the output of the <command>virsh capabilities</command> command. When deactivated, the CPU model
```

File: xml/libvirt_configuration_gui.xml [hunk 3]
```diff
@@ -174,49 +174,44 @@
...
+    must be specified from the models available in the drop-down box.
```
Comment (tbazant) [inline xml/libvirt_configuration_gui.xml:190]:
```suggestion
    needs to be specified from the models available in the drop-down box.
```

File: xml/libvirt_configuration_gui.xml [hunk 4]
```diff
@@ -174,49 +174,44 @@
...
+   </para>
+   <para>
+    The host CPU model generally provides a good trade-off between CPU features
+    and ability to migrate the &vmguest; &libvirt; does not model every aspect
```
Comment (tbazant) [inline xml/libvirt_configuration_gui.xml:194]:
```suggestion
    and ability to migrate the &vmguest;. &libvirt; does not model every aspect
```

File: xml/libvirt_configuration_gui.xml [hunk 5]
```diff
@@ -174,49 +174,44 @@
...
+    But the ABI provided to the &vmguest; is reproducible and during migration
+    the complete CPU model definition is transferred to the destination &vmhost;,
+    ensuring the migrated &vmguest; will see the exact same CPU model on the
+    destination.
+   </para>
+   <para>
+    The host-passthrough model provides the &vmguest; with a CPU that is exactly
+    the same as the &vmhost; CPU. This can be useful when the &vmguest; workload
+    requires CPU features not available in &libvirt;'s simplified host-model CPU.
+    The host-passthrough model is also required in some cases, e.g. when running
```
Comment (tbazant) [inline xml/libvirt_configuration_gui.xml:205]:
```suggestion
    The host-passthrough model is also required in some cases, for example, when running
```

File: xml/libvirt_configuration_gui.xml [hunk 6]
```diff
@@ -174,49 +174,44 @@
...
+    &vmguest;s with more than 4TB of memory. The host-passthrough model comes
+    with the disadvantage of reduced migration capability. A &vmguest; with
+    host-passthrough model CPU can only be migrated to a &vmhost; with identical
+    hardware.
+   </para>
+   <para>
+    For more information on &libvirt;'s CPU model and topology options, see
+    the <citetitle>CPU model and topology</citetitle> documentation
+    at <link xlink:href="https://libvirt.org/formatdomain.html#cpu-model-and-topology"/>.
```
Comment (tbazant) [inline xml/libvirt_configuration_gui.xml:214]:
```suggestion
   <xref linkend="libvirt-config-virsh-cpu"/>.
```

File: xml/libvirt_configuration_gui.xml [hunk 7]
```diff
@@ -174,49 +174,44 @@
...
+    requires CPU features not available in &libvirt;'s simplified host-model CPU.
+    The host-passthrough model is also required in some cases, e.g. when running
+    &vmguest;s with more than 4TB of memory. The host-passthrough model comes
+    with the disadvantage of reduced migration capability. A &vmguest; with
+    host-passthrough model CPU can only be migrated to a &vmhost; with identical
+    hardware.
+   </para>
+   <para>
+    For more information on &libvirt;'s CPU model and topology options, see
+    the <citetitle>CPU model and topology</citetitle> documentation
```
Comment (tbazant) [inline xml/libvirt_configuration_gui.xml:213]:
```suggestion
    For more information on &libvirt;'s CPU model and topology options, refer to
```

**Review by tbazant — CHANGES_REQUESTED** [review]
Good job! Minor changes only, and... https://github.com/SUSE/doc-sle/pull/932 needs to be merged first in order for the changed link to work

File: xml/libvirt_configuration_gui.xml [hunk 8]
```diff
@@ -174,49 +174,44 @@
...
+    The host CPU model generally provides a good trade-off between CPU features
+    and ability to migrate the &vmguest; &libvirt; does not model every aspect
+    of each CPU, so the &vmguest; CPU will not match the &vmhost; CPU exactly.
+    But the ABI provided to the &vmguest; is reproducible and during migration
+    the complete CPU model definition is transferred to the destination &vmhost;,
+    ensuring the migrated &vmguest; will see the exact same CPU model on the
+    destination.
+   </para>
+   <para>
+    The host-passthrough model provides the &vmguest; with a CPU that is exactly
```
Comment (tbazant) [inline xml/libvirt_configuration_gui.xml:202]:
```suggestion
    The <literal>host-passthrough</literal> model provides the &vmguest; with a CPU that is exactly
```

File: xml/libvirt_configuration_gui.xml [hunk 9]
```diff
@@ -174,49 +174,44 @@
...
+    the same as the &vmhost; CPU. This can be useful when the &vmguest; workload
+    requires CPU features not available in &libvirt;'s simplified host-model CPU.
```
Comment (tbazant) [inline xml/libvirt_configuration_gui.xml:204]:
```suggestion
    requires CPU features not available in &libvirt;'s simplified <literal>host-model</literal> CPU.
```

File: xml/libvirt_configuration_gui.xml (hunk 5 above)
Comment (tbazant) [reply xml/libvirt_configuration_gui.xml:205]:
```suggestion
    The <literal>host-passthrough</literal> model is also required in some cases, for example, when running
```

File: xml/libvirt_configuration_gui.xml [hunk 10]
```diff
@@ -174,49 +174,44 @@
...
+    The host-passthrough model is also required in some cases, e.g. when running
+    &vmguest;s with more than 4TB of memory. The host-passthrough model comes
+    with the disadvantage of reduced migration capability. A &vmguest; with
+    host-passthrough model CPU can only be migrated to a &vmhost; with identical
```
Comment (tbazant) [inline xml/libvirt_configuration_gui.xml:208]:
```suggestion
    &vmguest;s with more than 4TB of memory. The <literal>host-passthrough</literal> model comes
    with the disadvantage of reduced migration capability. A &vmguest; with
    <literal>host-passthrough</literal> model CPU can only be migrated to a &vmhost; with identical
```

File: xml/libvirt_configuration_gui.xml [hunk 11]
```diff
@@ -274,6 +269,13 @@
      </listitem>
     </varlistentry>
    </variablelist>
+   <important>
+    <title>Large memory &vmguest;s</title>
+    <para>
+      &vmguests; with memory requirements of 4TB or more must currently
+      use the host-passthrough CPU model.
```
Comment (tbazant) [inline xml/libvirt_configuration_gui.xml:276]:
```suggestion
      &vmguests; with memory requirements of 4TB or more currently need to
      use the <literal>host-passthrough</literal> CPU model.
```

**Review by tbazant — CHANGES_REQUESTED** [review]
another portion of minor updates

**Pull request merged**